cni/network/network.go

```diff
@@ -367,10 +367,13 @@ func (plugin *netPlugin) Add(args *cniSkel.CmdArgs) error {
 			return err
 		}
 
-		nwDNSInfo, err := getNetworkDNSSettings(nwCfg, result, k8sNamespace)
-		if err != nil {
-			err = plugin.Errorf("Failed to getDNSSettings: %v", err)
-			return err
+		nwDNSInfo := getCustomDNS(nwCfg)
```

Comment:
can we move all the dns related code to one function and return nwDnsInfo. move that function to network_windows.go.